hypertrace-graphql-labels-schema-api/src/main/java/org/hypertrace/graphql/label/schema/rule/StringCondition.java

```diff
@@ -9,12 +9,14 @@ public interface StringCondition {
   String TYPE_NAME = "LabelApplicationStringCondition";
 
   String OPERATOR_KEY = "operator";
-  String VALUE_KEY = "value";
+  String STRING_CONDITION_VALUE_KEY = "stringConditionValue";
```

Comment:
Please revert this, it's a breaking change. Instead, accept a list of values as a sibling to existing value and remove the non null on `value` (still technically breaking, but less critical as it'd only break on trying to read a new value with an old query).

You may also want to include a nullable enum that specifies which field is set - your `StringConditionValueType`. It should always be set on return, but on request you would need to default it to the singular form for backwards compat. Hope that makes sense